spec/util/object_storage/miq_swift_storage_spec.rb

```diff
@@ -0,0 +1,68 @@
+require "util/object_storage/miq_swift_storage"
+
+describe MiqSwiftStorage do
+  context "public methods" do
+    let(:uri) { "swift://foo.com/abc/def" }
+    let(:object_storage) { described_class.new(:uri => uri, :username => 'user', :password => 'pass', :region => 'region') }
```

Comment:
Why is `region` passed in here? Shouldn't it be a part of the uri?

Reply:
If it's not being used it looks like the `let(:object_storage)` will be the same across all the contexts so it can only be defined once.

Reply:
Region was left over from previous test.  :object_storage changes with every change to the :uri so it can't be defined only once.

Reply:
Yes, it can. `let` is lazily evaluated so uri will be used from the context in which `object_storage` is called. See the rspec documentation [here](https://relishapp.com/rspec/rspec-core/v/3-8/docs/helper-methods/let-and-let)

Reply:
Well then.  Changing.  And thank you for providing the "something new" that I learned today.   
:-)